### PR TITLE
Save editor hidden status to Playroom url

### PR DESCRIPTION
**Pull request by felixhabib**
See changeset

Last feature I want to get in before cutting a release

### CODE REVIEW

File: cypress/e2e/urlHandling.cy.ts [hunk 1]
```diff
@@ -30,6 +30,22 @@ describe('URL handling', () => {
...
+      cy.get('textarea').should('not.be.focused');
+
+      // Todo - write a test that checks the CodeMirror element is not visible
+      /*
+      The CodeMirror element is not visible, but it is in the DOM
+      This test fails because the element doesn't meet Cypress's requirements for being hidden
+      https://docs.cypress.io/guides/core-concepts/interacting-with-elements#An-element-is-considered-hidden-if
+      */
+      // cy.get('.CodeMirror-code').should('be.hidden');
+    });
```
Comment (felixhabib) [inline cypress/e2e/urlHandling.cy.ts:48]: This is proving very difficult to test. I don't think it is worth holding up the release unless there is an easy fix

File: cypress/e2e/urlHandling.cy.ts [hunk 2]
```diff
@@ -30,6 +30,22 @@ describe('URL handling', () => {
...
     });
+
+    it('editor hidden', () => {
+      cy.visit(
+        'http://localhost:9000/#?code=N4IgpgJglgLg9gJwBJQhMA7EAuGCCuYAvkA'
+      );
+
+      cy.get('textarea').should('not.be.focused');
+
+      // Todo - write a test that checks the CodeMirror element is not visible
```
Comment (askoufis) [inline cypress/e2e/urlHandling.cy.ts:41]: Feel like this todo is redundant given the explanation below.

File: cypress/e2e/urlHandling.cy.ts [hunk 3]
```diff
@@ -30,6 +30,22 @@ describe('URL handling', () => {
...
+      /*
+      The CodeMirror element is not visible, but it is in the DOM
+      This test fails because the element doesn't meet Cypress's requirements for being hidden
```
Comment (askoufis) [inline cypress/e2e/urlHandling.cy.ts:44]: Could be worth clarifying that we don't know what's causing it to fail the hidden requirement either.

**Review by askoufis — APPROVED** [review]

File: cypress/e2e/urlHandling.cy.ts (hunk 2 above)
Comment (felixhabib) [reply cypress/e2e/urlHandling.cy.ts:41]: I think I'll keep the todo only but remove the surrounding comment, so when a fix is found it prompts us to change it for both tests

**Pull request merged**